react-components/src/components/AuthBar/container.js

```diff
@@ -21,7 +21,7 @@ import { useNavigationContext } from '../../context/NavigationContext';
 import classes from './container.css';
 
 const Container = () => {
-    const container = document.querySelector('#miniaccount');
+    const container = document.querySelector('aside.navigation__root #miniaccount');
```

Comment:
Same like above :)

Reply:
Hi @herzog31, same here, the query selector is now passed via `<CommerceApp>`, please review

Reply:
Looks much better :)